src/components/Link/Link.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+
+export default class Link extends React.Component {
+  render() {
+    let download = '';
+    if (this.props.download) {
+      download = this.props.downloadText ? this.props.downloadText : '';
+    }
+    const help = this.props.helpIcon
+      ? (
+        <chi-icon
+          icon="circle-question-outline"
+          onClick={this.props.clickHelp}
+          onMouseEnter={this.props.mouseoverHelp}
+          onMouseLeave={this.props.mouseleaveHelp}>
+        </chi-icon>
+      )
+      : <chi-icon></chi-icon>;
+    return (
+      <chi-link
+        href="#"
+        disabled={this.props.disabled}
+        cta={this.props.cta}
+        target={this.props.target}
+        rel={this.props.rel}
+        hreflang={this.props.hreflang}
+        size={this.props.size}
+        download={this.props.download ? download : null}
+        alternative-text={this.props.alternativeText}>
+        {help}
+        <span>{this.props.title ? this.props.title : ''}</span>
+      </chi-link>
+    );
+  }
+}
+
+Link.propTypes = {
+  disabled: PropTypes.bool,
+  cta: PropTypes.bool,
+  target: PropTypes.oneOf(['_self', '_blank', '_parent', '_top', 'framename']),
```

Comment:
I think prototypes are not going to need target prop. Could we add a `click`interaction instead?

`click: PropTypes.func,`

With this the designers will be able to set whatever the interaction is necessary for their prototype cases